addons/mail/models/discuss/res_users.py

```diff
@@ -51,7 +51,7 @@ def _init_store_data(self, store):
             "Store": {
                 "hasGifPickerFeature": get_param("discuss.tenor_api_key"),
                 "hasMessageTranslationFeature": get_param("mail.google_translate_api_key"),
-                "channel_types_with_seen_infos": self.env["discuss.channel"]._types_allowing_seen_infos(),
+                "channel_types_with_seen_infos": self.env["discuss.channel"]._types_allowing_seen_infos().sort(),
```

Comment:
`.sort()` is in-place, doesn't return the value. Maybe you'd want `sorted()` standalone